src/main/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModule.java

```diff
@@ -206,6 +229,34 @@ public void buildComplete(BuildCompleteEvent event) {
         criticalPathComputer = null;
       }
       profilePath = null;
+      cpuUserTimeForActions = Duration.ofMillis(0);
```

Comment:
It's better to initialise this variables on other method, e.g. on `executorInit`.

Reply:
Could you move please this inititalization on `executorInit`?

Reply:
I assume that you could move this to `executorInit` and call this method on `setUp()` on tests.

Reply:
I would also prefer having these on `executorInit` or some other part of startup, to be sure that the values are zeroed out regardless of how the previous build ended.

Reply:
Zeroing them on startup would also make it clearer that they are properly initialized.